subprojects/core/src/test/groovy/org/gradle/groovy/scripts/internal/BuildScopeInMemoryCachingScriptClassCompilerTest.groovy

```diff
@@ -1,131 +0,0 @@
-/*
- * Copyright 2010 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.gradle.groovy.scripts.internal
-
-import org.gradle.api.Action
-import org.gradle.api.internal.initialization.loadercache.ClassLoaderId
-import org.gradle.groovy.scripts.Script
-import org.gradle.groovy.scripts.ScriptSource
-import org.gradle.groovy.scripts.TestScript
-import org.gradle.groovy.scripts.Transformer
-import spock.lang.Specification
-
-class BuildScopeInMemoryCachingScriptClassCompilerTest extends Specification {
```

Comment:
⭕️ Can you make sure that these things are still covered by other tests? I.e. we make sure that the cross-build cache works and we don't recompile all the time?

Reply:
This should be covered by `CrossBuildScriptCachingIntegrationSpec`